docs/getting-started/ios/setup.md

```diff
@@ -32,11 +32,14 @@ The most interesting file is `fastlane/Fastfile`, which contains all the informa
 
 ## What's next?
 
-_fastlane_ created all the required files for you, now you can go ahead and customise it to generate screenshots or to automatically distribute new builds
+_fastlane_ created all the required files for you. Now you can go ahead and customise _fastlane_ to generate screenshots, or automatically distribute new builds, and much, much more. Here're some examples:
 
 - [Generate localized iOS screenshots for the App Store](screenshots.md)
 - [Automatic iOS Beta deployment](beta-deployment.md)
 - [Automatic iOS App Store deployment](appstore-deployment.md)
+- Discover all [_fastlane_ actions](/actions)
+
+Do note that if the automation you're building requires connectivity with Apple's servers, such as for code signing when building your app, or uploading your app to the App Store Connect, and so on, you will need to authenticate. Check out [Authenticating with Apple services](authentication.md) to learn the best ways to authenticate, catered for your specific use case.
```

Comment:
```suggestion
Do note that if the automation you're building requires connectivity with Apple's servers, such as for code signing when building your app, or uploading your app to the App Store Connect, and so on, you will need to authenticate. Check out [Authenticating with Apple services](/getting-started/ios/authentication) to learn the best ways to authenticate, catered for your specific use case.
```

This this link to `/getting-started/ios/authentication` instead of the `.md` file? 🤔

Reply:
@joshdholtz I just followed the same syntax of the bullet points up there:

```md
- [Generate localized iOS screenshots for the App Store](screenshots.md)
- [Automatic iOS Beta deployment](beta-deployment.md)
- [Automatic iOS App Store deployment](appstore-deployment.md)
```

As these are all in the same-level directory, redirects like these are 👌 

Do I need to change this?